### ISSUE TITLE
negative scores?

### ISSUE THREAD

**Issue by kkmann** [pharmaR/riskmetric#271]
When I run

```
pkg_ref(c("riskmetric", "utils")) %>%
+     pkg_assess() %>%
+     pkg_score()
```

on the current CRAN version, I get negative `pkg_scores`

```
# A tibble: 2 x 19
  package    version pkg_ref             pkg_score news_current has_vignettes
  <chr>      <chr>   <lst_f_p_>              <dbl> <pkg_scor>   <pkg_scor>   
1 riskmetric 0.1.2   riskmetric<install>     -1.57 1            1            
2 utils      4.1.2   utils<install>         -15.9  0            1            
# ... with 13 more variables: has_bug_reports_url <pkg_scor>,
#   bugs_status <pkg_scor>, license <pkg_scor>, export_help <pkg_scor>,
#   downloads_1yr <pkg_scor>, has_website <pkg_scor>, r_cmd_check <pkg_scor>,
#   remote_checks <pkg_scor>, has_maintainer <pkg_scor>,
#   exported_namespace <pkg_scor>, has_news <pkg_scor>,
#   has_source_control <pkg_scor>, covr_coverage <pkg_scor>
```

Is this a bug or by design?

Tagging @dgkf and @Xyarz.

**Comment by borgmaan** [comment]
@kkmann -- we noticed similar behavior that led to reporting #270. It seems like a bug in the CRAN release. It has been patched on `master` and the team discussed making a new CRAN release with the patch at the last sprint meeting. Should be coming soon.

**Comment by kkmann** [comment]
Thanks!, Seems to work. It is a bit odd, that vignettes are missing then (not build per default) but I guess that comes from judging the installed package, not its sources

```
> pkg_ref(c("riskmetric", "utils")) %>%
+     pkg_assess()
# A tibble: 2 x 21
  package    version pkg_ref             news_current has_vignettes size_codebase
  <chr>      <chr>   <lst_f_p_>          <lst_f_p_>   <lst_f_p_>    <lst_f_p_>   
1 riskmetric 0.1.2   riskmetric<install> TRUE         0             <error>      
2 utils      4.1.2   utils<install>      <lgl [0]>    1             <error>      
# ... with 15 more variables: has_bug_reports_url <lst_f_p_>,
#   bugs_status <lst_f_p_>, license <lst_f_p_>, export_help <lst_f_p_>,
#   reverse_dependencies <lst_f_p_>, downloads_1yr <lst_f_p_>,
#   dependencies <lst_f_p_>, has_website <lst_f_p_>, r_cmd_check <lst_f_p_>,
#   remote_checks <lst_f_p_>, has_maintainer <lst_f_p_>,
#   exported_namespace <lst_f_p_>, has_news <lst_f_p_>,
#   has_source_control <lst_f_p_>, covr_coverage <lst_f_p_>
```

**Comment by emilliman5** [comment]
A new version is available on CRAN (v0.2.0) with a fix.
